content/en/blog/news/2022-gsoc-work-report.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Jenkins X building a new UI"
+date: 2022-11-13
+draft: false
+description: This project replaces the older UI with the new UI with more powerful features.
+categories: [blog]
+keywords: [Community, jenkins-x, gsoc, 2022]
+slug: "GSoC-2022-work-report"
+aliases: []
+author: Rajat Gupta
+---
+
+# Jenkins X
+Jenkins X is an open-source solution that provides automated continuous integration and continuous delivery (CI/CD) and automated testing tools for cloud-native applications on Kubernetes.
+
+## Why need a new UI?
+A good UI is an essential for a CI/CD tool, as not everyone is familiar with the CLI.
+
+The current UI (jx-pipeline-visualizer) is a read-only UI, user can view the logs of PipelineActivity but can not start nor stop the pipeline.
+
+Features that UI will provide: 
+- Start and Stop a PipelineActivity.
+- Have a audit trail.
+- A DAG representation of PipelineActivity.
+- RBAC to prevent the distribution of cluster access.
+
+New Jenkins X UI focus on Simplicity, Security and a Superb User Experience.
+
+This is NOT GA(General Availability) yet.
+
+Visit the project repo [here](https://github.com/jenkins-x/jx-ui) to try it.
+
+## Work Done
+### Stop a running or pending PipelineActivity
```

Comment:
Remove this section? Is this really under what has bben done? It shows the "jx" command

Reply:
Yes, this is done, we can now stop a pipeline from UI, I'll rewrite the explanation.